### PR TITLE
Enumerable

### PR DESCRIPTION
**Pull request by AndresFMoya**

### CODE REVIEW

File: enumerable.rb [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+module Enumerable
+  def my_each
```
Comment (bolah2009) [inline enumerable.rb:4]: #### `REQUIRED`

:bug: There's no return value. Also, let's handle cases where a block is given.

```ruby
irb(main):001:0> require_relative 'enumerable'
=> true
irb(main):002:0> [1,2,3,4,5].each{|i| i > 2}
=> [1, 2, 3, 4, 5]
irb(main):003:0> [1,2,3,4,5].my_each{|i| i > 2}
=> nil
```

#### Do this for all other methods. :blush:

**Review by bolah2009 — CHANGES_REQUESTED** [review]
# Hi @AndresFMoya :wave: 

Please, test all methods with the built-in methods as shown in the comment below, currently, most methods are not working.

You can find expected behaviors of this methods [here](https://ruby-doc.org/core-2.6.3/Enumerable.html)

Fix this and submit another code review request. :ok_hand: 

#### Feel free to reach me on slack if you having issues implementing these features. :+1: 

Good luck with your next submission. :star2: 

Cheers
[Bola](https://bolabuari.com/)

File: enumerable.rb (hunk 1 above)
Comment (AndresFMoya) [reply enumerable.rb:4]: Hi, @bolah2009. Thanks for pointing me the right address. The methods were corrected and compared with the built-in methods. I am awaiting further revision. 

Regards!
**Andres**

**Review by bellom — APPROVED** [review]
Hi @AndresFMoya,
**Great work making all the necessary corrections from the previous review.**
## Congratulation :+1:  
_Go ahead and merge to master and don't forget to submit project completion form.
Goodluck in your next project :slightly_smiling_face:_

**Pull request merged**